### PR TITLE
Copy/Past text should include opacity and tracking - edge cases

### PR DESCRIPTION
**Pull request by shaoshing**
These two styles are included in the previous PR, and this PR covers some edge cases:

1. copy/paste should include 0 tracking.
2. opacity is not copied after changing. Addresses #3100 
3. bonus: fix changing type weight.

### CODE REVIEW

File: src/js/stores/font.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ define(function (require, exports, module) {
                 };
             }
 
-            if (textStyle.tracking) {
+            if (Number.isFinite(textStyle.tracking)) {
```
Comment (shaoshing) [inline src/js/stores/font.js]: ```!!0 === false``` cause 0 tracking is been left behind.

File: src/js/jsx/sections/style/Type.jsx [hunk 2]
```diff
@@ -157,6 +157,26 @@ define(function (require, exports, module) {
...
             }
 
+            this._setPostScript(postScriptName);
+        },
+        
+        /**
+         * Handle change of type weight.
+         *
+         * @private
+         * @param  {string} postScriptName
```
Comment (iwehrman) [inline src/js/jsx/sections/style/Type.jsx]: Extra space here after `@param`

File: src/js/stores/font.js (hunk 1 above)
Comment (iwehrman) [inline src/js/stores/font.js]: :person_frowning:

**Comment by iwehrman** [comment]
I set all the properties on a type layer and then sampled it, but only a few made it over:
img

**Comment by chadrolfs** [comment]
@iwehrman it looks like you're using the Sampler.  #3100 is specific to copy/pasting via the panel icons.  I think click with Sampler just copies fill color.

**Comment by iwehrman** [comment]
Yes, sorry. I was confused. In this case at least it seems to be working completely correctly.

**Comment by iwehrman** [comment]
After discussion with @shaoshing and @placegraphichere, I'm satisfied. I'll wait to get a thumbs-up from @chadrolfs before merging.

**Comment by iwehrman** [comment]
@chadrolfs gives us :+1: :+1:, so bombs away!

**Pull request merged**